### DISCUSSION TITLE
Error while fetching extensions. XHR failed

### DISCUSSION THREAD

**Discussion by han-jiangxue** [microsoft/vscode-discussions#624] [Extension Development QnA]
When I'm trying to get any extension, there comes a waring "Error while fetching extensions. XHR failed". And after opening the Toggle Developer Tools, it shows "POST https://marketplace.visualstudio.com/_apis/public/gallery/extensionquery net::ERR_CERT_AUTHORITY_INVALID". Also, I can't use the button "check for updates...", same error prompt of "net::ERR_CERT_AUTHORITY_INVALID".

I tried the following but it didn't work：
1. Check the system's proxy settings and make sure they are configured correctly；
2. Open VSCode and press F1 to open the Command Palette. Type user settings, and then press Enter. In the search bar, type proxy and press Enter. Look for the setting labeled Http: Proxy Support and change its value to Override；
3. Uninstall vscode, delete the relevant configuration, and then reinstall it.

About: Windows x64 1.78.2

Is there any other solutions?

**Comment by alefragnani** [comment]
Hi @han-jiangxue ,

If you have this issue while _installing_ an extension, I suggest you to take a look for similar issues at VS Code repository. I found a few with references to `ERR_CERT_AUTHORITY_INVALID` (https://github.com/microsoft/vscode/issues?q=is%3Aissue+ERR_CERT_AUTHORITY_INVALID+is%3Aopen+). Maybe you could find the fix in one of them. Otherwise, add a new one with your scenario and details. 

The VS Code repository is the best location to report issues on VS Code. This Discussions repository, on the other hand, is focused for extension development for now.

Hope this helps

**Reply by han-jiangxue** [reply]
Thanks for the reply, it was my mistake. I eventually solved the problem by resetting my PC.
